morpheushelper/cogs/rules.py

```diff
@@ -16,6 +16,18 @@ class RulesCog(Cog, name="Rule Commands"):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    @staticmethod
+    async def get_message_cancel(bot: Bot, channel: TextChannel, member: Member) -> [Tuple[Optional[str],
+                                                                                           List[Optional[File]]]]:
+        content, files = await read_normal_message(bot, channel, member)
+        if content == translations.cancel:
+            embed = Embed(title=translations.rule, colour=Colours.RuleCommands,
+                          description=translations.msg_send_cancel)
+            await channel.send(embed=embed)
+            return None, []
```

Comment:
shouldn't that be a tuple and a list if you watch the typing of the method? 

```python
return (), []
```

Reply:
Yeah, my typing was a bit wrong, I fixed that now